src/moh_togo_uat/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<string name="app_name">MOH-TOGO UAT</string>
```

Comment:
Is this for user acceptance testing? You might consider using the generic Medic app if it is for UAT only?  We typically don't build APKs for UAT - just training/production.